addons/website_blog/tests/test_website_blog_flow.py

```diff
@@ -108,3 +108,10 @@ def test_blog_comment(self):
 
         self.assertFalse(self.env['mail.message'].sudo().search(
             [('model', '=', 'blog.post'), ('attachment_ids', 'in', second_attachment.ids)]))
+
+    def test_website_blog_teaser_content(self):
+        """ Make sure that the content of the post is correctly rendered in proper plain text. """
```

Comment:
```suggestion
        """ Make sure that the content of the post is correctly rendered in
             proper plain text. """
```

Keep docstring under 80 char per line, see https://github.com/odoo/odoo/pull/83508#discussion_r796442104

Basically, any of:
```
def blabla1():
    """
    Fixes the id fields in import and exports, and splits field paths
    on '/'.
    """

def blabla2():
    """ Fixes the id fields in import and exports, and splits field
    paths on '/'.
    """

def blabla3():
    """ Fixes the id fields in import and exports, and splits field
        paths on '/'.
    """
```
https://www.python.org/dev/peps/pep-0257/#handling-docstring-indentation](https://www.python.org/dev/peps/pep-0257/#handling-docstring-indentation
AFAIK, do whatever you want with the first line, but for following lines (2 and +) indent it the same way.